source/developers/how-tos/enable-python-upgrade-automation.rst

```diff
@@ -0,0 +1,44 @@
+How To Enable Python Upgrade Automation
+#######################################
+
+This How-to will walk you through the process of getting the automated python
+requirements upgrade jobs setup on a repository.
+
+Assumptions
+***********
+
+* You know how to use Git and GitHub PR Workflows
+
+* You have ``admin`` access to the repository.
+
+Steps
+*****
+
+1. Replace ``<repo_name>`` with the name of your repository in the following URL
+   and go to it:
+
+   .. code::
+
+      https://github.com/openedx/<repo_name>/actions/new?category=owner
+
+2. Click :guilabel:`Configure` on the ``Python Requirements Upgrade Workflow``
+   card.
+
+   .. image:: /_images/developer_how_tos/python-requirements-upgrade-workflow.png
+      :alt: An image of the Python Requiremnets Upgrade Workflow Card in Github's UI.
+
+3. Review the contets of the github action, make any changes that you might want
+   to make.  If you're not sure, the defaults work fine for most use cases.
+
+4. Commit the changes to a branch and get the PR reviewed and merged.
```

Comment:
Maybe we should add a bit more specifics, or a screen shot, here as this isn't the default path.

![image](https://user-images.githubusercontent.com/1270274/195675616-2c0aa885-3d2b-435a-859d-400e2ba6d9fc.png)